.golangci.yml

```diff
@@ -34,8 +34,8 @@ linters:
     - unused
     - unparam
     - misspell
-    - wsl
-    - nolintlint
+    # - wsl
+    # - nolintlint
```

Comment:
disabling these as they have are not completed and are causing no linting to be run currently and crashing this linter because of the errors

Reply:
hold on, please try increase the timeout. Let's see how long it takes

Reply:
its not failing due to timeout, there are 100+ errors atributed to these two linters so it crashes due to linting errors